packages/create-snowpack-app/index.js

```diff
@@ -151,9 +157,16 @@ if (requiredVersion < 10) {
     cwd: targetDirectory,
     stdio: "inherit",
   };
-  const npmInstallProcess = useYarn
-    ? execa("yarn", ["--silent"], npmInstallOptions)
-    : execa("npm", ["install", "--loglevel", "error"], npmInstallOptions);
+  function installProcess(packageManager) {
+    return packageManager === "npm"
```

Comment:
NIT: would a switch case be better here instead of a nested ternary?

Reply:
@stramel On refelection I agree, I have made the change.